SleepEngine/Engine/Components/HierarchyNode.cpp

```diff
@@ -1,6 +1,6 @@
 #include "stdafx.h"
 #include "HierarchyNode.h"
-#include <Engine/Utils/stl_utils.h>
+#include <Engine/Utils/stl_utils.h>

```

Comment:
Note for Taras: Yeah, a lot of strange changes without changes. This is because some of your files contain non CLRF line endings and my IDE forces me to change your's 'non-reliable' line changes to mine. Please, let's just use CLRF line changes always.